### PR TITLE
1723 n xmonpd

### LINKED ISSUE #1191: Export CanSAS data format
This is subset of Nexus ... for SAXS data. Seams reasonable.
http://download.nexusformat.org/doc/html/classes/applications/NXcanSAS.html

### PR DESCRIPTION
**Pull request by kif**
close #1191

### CODE REVIEW

File: pyFAI/io/nexus.py [hunk 1]
```diff
@@ -744,7 +744,7 @@ def save_NXcansas(filename, result,
             errors_ds.attrs["doc"] = "standard error of the mean"
             errors_ds.attrs["interpretation"] = "spectrum"
             errors_ds.attrs["units"] = "arbitrary"
-            # integration_data["errors"] = errors_ds
+            integration_data["errors"] = errors_ds
```
Comment (woutdenolf) [inline pyFAI/io/nexus.py:747]: Small nit: use a softlink instead of copying the data

**Review by woutdenolf — APPROVED** [review]
Apart from small nit about the softlink, LGTM

File: pyFAI/io/nexus.py (hunk 1 above)
Comment (kif) [reply pyFAI/io/nexus.py:747]: this does not duplicate (copy) the data, but makes a hard-link to the data.
https://docs.h5py.org/en/stable/high/group.html#hard-links

Comment (kif) [reply pyFAI/io/nexus.py:747]: Maybe I'll go for "I_errors" since I dislike the name "errors"

**Pull request merged**